doc/source/write-documentation.rst

```diff
@@ -1,4 +1,4 @@
-Writing documentation
+How to write documentation
 =====================
```

Comment:
The `===` on this line should be as long as the line above

Reply:
Oh I get it now. It was mentioned in the doc but didn't quite get it.I will make required changed and push it.